pkg/tinydtls/Makefile.include

```diff
@@ -2,6 +2,8 @@ PKG_BUILDDIR ?= $(PKGDIRBASE)/tinydtls
 
 INCLUDES += -I$(PKG_BUILDDIR)
 
+CFLAGS += -Wno-implicit-fallthrough
```

Comment:
Doesn't this disable the warnings for all files when USEPKG contains tinydtls?

Reply:
Does it work if the CFLAGS are modified in Makefile instead of Makefile.include?

Reply:
Ups, I overlooked the .include.

Reply:
you're right - for both. It also works when putting this into Makefile. But then we should likely move the llvm CFLAGs additions there, too? But in a separate PR, I guess.